source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -1253,6 +1345,159 @@ TEST_F(NiSyncDriver6683Test, GetTimeReferenceNames_ReturnsSuccess)
   EXPECT_EQ(VI_SUCCESS, viStatus);
 }
 
+TEST_F(NiSyncDriver6683Test, EnableTimeStampTrigger_ReturnsSuccess)
+{
+  ViStatus viStatusEnable;
+  auto grpcStatusEnable = call_EnableTimeStampTrigger(
+     NISYNC_VAL_PFI1, // terminalName
+     NISYNC_VAL_EDGE_RISING, // activeEdge
+     &viStatusEnable);
+  EXPECT_TRUE(grpcStatusEnable.ok());
+  EXPECT_EQ(VI_SUCCESS, viStatusEnable);
+
+  ViStatus viStatusDisable;
+  auto grpcStatusDisable = call_DisableTimeStampTrigger(
+     NISYNC_VAL_PFI1,
+     &viStatusDisable);
+  EXPECT_TRUE(grpcStatusDisable.ok());
+  EXPECT_EQ(VI_SUCCESS, viStatusDisable);
+}
+
+TEST_F(NiSyncDriver6683Test, EnableTimeStampTriggerWithInvalidTerminal_ReturnsError)
+{
+  ViStatus viStatus;
+  auto grpcStatus = call_EnableTimeStampTrigger(
+     NISYNC_VAL_GND, // terminalName
+     NISYNC_VAL_EDGE_RISING, // activeEdge
+     &viStatus);
+
+  EXPECT_TRUE(grpcStatus.ok());
+  EXPECT_EQ(NISYNC_ERROR_TERMINAL_INVALID, viStatus);
+}
+
+TEST_F(NiSyncDriver6683Test, DisableTimeStampTriggerNotReserved_ReturnsError)
+{
+  ViStatus viStatus;
+  auto grpcStatus = call_DisableTimeStampTrigger(
+     NISYNC_VAL_PFI1, // terminalName
+     &viStatus);
+
+  EXPECT_TRUE(grpcStatus.ok());
+  EXPECT_EQ(NISYNC_ERROR_RSRC_NOT_RESERVED, viStatus);
+}
+
+TEST_F(NiSyncDriver6683Test, ReadTriggerTimeStamp_WhenNoTriggerOccurs_ReturnsTimeoutError)
```

Comment:
The auto tests should follow a "given", "when", "then" pattern, so this would probably be named `NoTriggerOccurs_ReadTriggerTimeStamp_ReturnsTimeoutError`, right? Then you'd put a blank line before the `EXPECT_TRUE` at the end.

Same below.

Reply:
I named and refactored the bottom three tests to follow that pattern.